portfolio/src/main/webapp/leave-comment/script.js

```diff
@@ -1,8 +1,56 @@
+function onSignIn(googleUser) {
+  if ($('#sign-out').length != 0) {
+    return;
+  }
+
+  let loggedUser = googleUser.getAuthResponse().id_token;
+  addUserTokenIdToForm(loggedUser);
+
+  $('#leave-comment')
+      .append(
```

Comment:
Wouldn't it be better to have the signOut button all the time in the DOM, but only toggle its visibility? Or, since you have multiple elements which need to be hidden or shown, you can toggle the visibility of a container which includes all of them? Or mark all the elements that should be visible only if the user signed in with a single class (e.g. `class=signedInOnly`) and select them using a class selector? Or maybe toggle a `signedIn` class on `<body>` and have a CSS definition which would make `signedInOnly` eleents be visible only if they are descendants of a `<body class=signedIn>`? Whoa, so many options to choose from.

Reply:
I agree with Michał.

Reply:
Done.